lib/netsparker/vulnerability.rb

```diff
@@ -146,6 +146,8 @@ def cleanup_html(source)
 
       result.gsub!(/<a href=\"(.*?)\" (.*?)>(.*?)<\/a>/i) { "\"#{$3.strip}\":#{$1.strip}" }
       result.gsub!(/<a href=\'(.*?)\'>(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
+      result.gsub!(/<a target=\'_blank\' href=\'(.*?)\'><i class=\'(.*?)\'><    \/i>(.*?)<\/a>/) { "\"#{$3.strip}\":#{$1.strip}" }
```

Comment:
@cbp1983 

- The extra space in `<    \/i>` breaks the regex for me.
- Pedantic point: use a non-capturing group. Instead of `<i class=\'(.*?)\'>` use `<i class=\'(?:.*?)\'>`. Then, use `$2` instead of `$3`. A non-capturing group (`(?:.*?)`) basically does the same thing as a capturing group (`(.*?)`) but it doesn't save it in a `$` variable. In this case, there's no big implication for using one or the other so you can ignore this comment if you want.

Reply:
Thank you!  Changes are implemented.  Don't know how the first error crept in, sorry.

For your second point, is there a reason why we take this approach on the new item but don't have that approach on line 147?

Reply:
No reason. It can be applied there too!